social-web-maintenance-wg-charter.html

```diff
@@ -190,6 +190,32 @@ <h3 id="out-of-scope">Out of Scope</h3>
           </p>
         </section>
 
+        <section id="section-workflow-for-active-items">
+          <h3 id="workflow-for-active-items">Workflow for Active Items</h3>
+          <p>
+            It is important to the health of the group and the quality of review
+            to avoid too much parallel work being scheduled concurrently. For
+            this reason, the Chair(s) may choose to stagger the scheduling of
+            kick-offs, active work, and review of work items, or even defer
+            final Chair and member review of items into a queue, if they feel
+            they cannot adequately supervise too many such processes in
```

Comment:
I'd probably just leave it at this:
```suggestion
            they cannot adequately supervise the requested work
```

removing the "parallel" on the next line